ui/admin/tests/acceptance/host-catalogs/read-test.js

```diff
@@ -60,25 +61,56 @@ module('Acceptance | host-catalogs | read', function (hooks) {
     await visit(urls.hostCatalogs);
     await a11yAudit();
     assert.strictEqual(currentURL(), urls.hostCatalogs);
-    await visit(urls.hostCatalog);
+
+    await click(`[href="${urls.hostCatalog}"]`);
     await a11yAudit();
+
     assert.strictEqual(currentURL(), urls.hostCatalog);
   });
 
   test('cannot navigate to a host catalog form without proper authorization', async function (assert) {
     assert.expect(1);
+    await visit(urls.projectScope);
     instances.hostCatalog.authorized_actions =
       instances.hostCatalog.authorized_actions.filter(
         (item) => item !== 'read'
       );
-    await visit(urls.hostCatalog);
-    assert.notOk(find('main tbody .rose-table-header-cell:nth-child(1) a'));
+
+    await click(`[href="${urls.hostCatalogs}"]`);
+
+    assert.dom(`[href="${urls.hostCatalog}"]`).doesNotExist();
   });
 
-  test('visiting an unknown host catalog displays 404 message', async function (assert) {
+  test('cannot navigate to host catalogs tab without proper authorization', async function (assert) {
     assert.expect(1);
+    await visit(urls.orgScope);
+    instances.scopes.project.authorized_collection_actions['host-catalogs'] =
+      [];
```

Comment:
is there a preference between using the filter method or setting the attribute equal to empty list ?

Reply:
I think the preference would be filter. By setting it to an empty list, we could get a false positive. Using the filter method allows us to be exact in what is being tested.

Reply:
I didn't realize this before, but authorized collections actions is already tested in the list-test.js files. So I can instead refactor those files to use the more appropriate methods.

Reply:
🤦🏼‍♂️ Thanks for noticing that before we added a bunch of duplicate tests.